.github/workflows/main.yml

```diff
@@ -36,7 +47,7 @@ jobs:
       - run: yarn install
       - run: yarn setup
       - run: yarn lint
-      - run: yarn test unit
+      - run: yarn test:unit
```

Comment:
Should we also separate this out to its own check?

Reply:
sure, yeah we could

Reply:
actually, probably not, seems like a waste of a runner, integration tests can get hefty hence their own runner